libgadget/blackhole.c

```diff
@@ -154,6 +225,13 @@ void set_blackhole_params(ParameterSet * ps)
 
         blackhole_params.BlackHoleFeedbackMethod = param_get_enum(ps, "BlackHoleFeedbackMethod");
         blackhole_params.BlackHoleRepositionEnabled = param_get_int(ps, "BlackHoleRepositionEnabled");
+        /***********************************************************************************/
+        blackhole_params.BH_DynFrictionMethod = param_get_int(ps, "BH_DynFrictionMethod");
+        blackhole_params.BH_DFBoostFactor = param_get_int(ps, "BH_DFBoostFactor");
+        blackhole_params.BH_DFbmax = param_get_int(ps, "BH_DFbmax");
```

Comment:
This should be param_get_double()

Reply:
I think it doesn't matter right now because the default variable is actually an integer, which will be cast to a double, but it could be confusing if someone tries to set it to 20.5 and nothing happens!